.github/workflows/functional_test.yaml

```
@@ -0,0 +1,104 @@
+name: Functional Test Helm Chart
+
+on: pull_request
+
+jobs:
+  e2e-test:
+    name: Functional Test Helm Chart - ${{ matrix.container_runtime }}
```

No need to repeat this in the job name. If we change to this:
```suggestion
    name: ${{ matrix.container_runtime }}
```
Github will show `Splunk Platform Functional Test / cri-o`